.evg.yml

```diff
@@ -713,11 +832,21 @@ functions:
         script: |
           ARTIFACTS_DIR=artifacts
           S3_ARTIFACTS_DIR='mongo-jdbc-driver/artifacts/${version_id}/${build_variant}'
+          export BUILD_TYPE="snapshot"
```

Comment:
Added this BUILD_TYPE variable to use for the tasks to determine whether the eap or standard build is running.